client/src/components/ChatSideBanner/ChatSideBanner.tsx

```diff
@@ -29,7 +30,9 @@ const ChatSideBanner = ({ loggedInUser }: Props): JSX.Element => {
   return (
     <Grid className={classes.chatSideBanner}>
       <Box className={classes.userPanel}>
-        <AvatarDisplay loggedIn user={loggedInUser} />
+        <Link href="/profile">
+          <AvatarDisplay loggedIn user={loggedInUser} />
+        </Link>
```

Comment:
The link you are using is not proper, this will cause refreshes in the app which means we need to load the entire app again. I would utilise the react-router-dom's `Link` component instead.